### ISSUE TITLE
Multiple bundle activators should be a build error

### ISSUE THREAD

**Issue by njbartlett** [bndtools/bnd#2667]
The `@Header` annotation can be used to generate the `Bundle-Activator` header, for example:

```java
@Header(name = Constants.BUNDLE_ACTIVATOR, value = "${@class}")
public class MyActivator implements BundleActivator { ... }
```

Unfortunately if we have multiple classes in the bundle so annotated, then `Bundle-Activator` is generated as a list, which is not legal in OSGi:

```
Bundle-Activator: org.example.Activator1, org.example.Activator2
```

Bnd should know that `Bundle-Activator` only permits a single entry, and raise a build-time error if multiple entries are found. It should also raise an error if a `Bundle-Activator` header is generated in an artifact that does not permit it, such as a Fragment or an Extension Bundle.

**Comment by stbischof** [comment]
it should be the same with some other constants, like
```
BUNDLE_SYMBOLICNAME
BUNDLE_NAME
BUNDLE_VERSION
EXTENSION_BUNDLE_ACTIVATOR

```

**Comment by bjhargrave** [comment]
Does the Verifier in Bnd not complain about these?

**Comment by bjhargrave** [comment]
Fixed in 4.2.
